seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/TaskExecutionService.java

```diff
@@ -177,20 +180,40 @@ public PassiveCompletableFuture<TaskExecutionState> deployLocalTask(
             submitThreadShareTask(executionTracker, byCooperation.get(true));
             submitBlockingTask(executionTracker, byCooperation.get(false));
             taskGroup.setTasksContext(taskExecutionContextMap);
-            executionContexts.put(taskGroup.getTaskGroupInfo(), new TaskGroupContext(taskGroup));
-            cancellationFutures.put(taskGroup.getTaskGroupInfo(), cancellationFuture);
+            executionContexts.put(taskGroup.getTaskGroupLocation(), new TaskGroupContext(taskGroup));
+            cancellationFutures.put(taskGroup.getTaskGroupLocation(), cancellationFuture);
         } catch (Throwable t) {
             logger.severe(ExceptionUtils.getMessage(t));
             resultFuture.completeExceptionally(t);
         }
+        resultFuture.whenComplete((r, s) -> {
+            InvocationFuture<Object> invoke = null;
+            long sleepTime = 1000;
```

Comment:
I suggest use `RetryUtils` to do retry.

Reply:
At present, RetryUtils cannot be used in this place. It should be judged by conditions to retry, rather than throwing exceptions.